docs/third-party/bunja.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: Bunja
+description: State Lifetime Manager
+nav: 5.03
+keywords: scope,di,raii,lifetime
+---
+
+[Bunja](https://github.com/disjukr/bunja) is lightweight State Lifetime Manager.
+
+It provides an RAII wrapper for jōtai atoms.
+
+---
+
+See also:
+
+- [Bunja README](https://github.com/disjukr/bunja/blob/main/README.md)
+- [Presentations](https://github.com/disjukr/bunja/tree/main/presentations)
+
+## install
+
+```
+npm install bunja
+```
+
+### Defining a Bunja
+
+You can define a bunja using the `bunja` function.
+
+When you access the defined bunja with the `useBunja` hook, a bunja instance is created.
+
+If all components in the render tree that refer to the bunja disappear, the bunja instance is automatically destroyed.
+
+If you want to trigger effects when the lifetime of a bunja starts and ends, you can use the `bunja.effect` field.
+
+```ts
+import { bunja } from 'bunja'
+import { useBunja } from 'bunja/react'
+
+const countBunja = bunja([], () => {
+  const countAtom = atom(0)
+  return {
+    countAtom,
+    [bunja.effect]() {
+      console.log('mounted')
+      return () => console.log('unmounted')
+    },
+  }
+})
+
+function MyComponent() {
+  const { countAtom } = useBunja(countBunja)
+  const [count, setCount] = useAtom(countAtom)
+  // Your component logic here
+}
+```
+
+### Defining a Bunja that relies on other Bunja
+
+If you want to manage a state with a broad lifetime and another state with a narrower lifetime, you can create a (narrower) bunja that depends on a (broader) bunja.
+
+For example, you can think of a bunja that manages the WebSocket connection and disconnection, and another bunja that subscribes to a specific resource over the connected WebSocket.
```

Comment:
Would it be possible to provide a simpler example to show the dependency case in this second section?

Reply:
Yes, replaced it with a modal screen example. https://github.com/pmndrs/jotai/pull/2938/commits/afcb0d87e601cd4ebe01eca6f62166ac2d83a626